hive_metastore_client/hive_metastore_client.py

```diff
@@ -212,3 +212,29 @@ def _validate_lists_length(list_a: List[Any], list_b: List[Any]) -> None:
                 "m=_validate_lists_length, msg=The length of the two provided "
                 "lists does not match"
             )
+
+    def get_partition_keys(self, db_name: str, table_name: str) -> List[FieldSchema]:
+        """
+        Gets the partition keys from a table.
+
+        An empty list will return when no table is found or
+        when the table has no partitions
+
+        :param db_name: database name where the table is at
+        :param table_name: table name which the partition keys belong to
+        """
+        table = self.get_table(dbname=db_name, tbl_name=table_name)
+        return list(table.partitionKeys) if table else []
```

Comment:
the `table.partitionKeys` isn't a list already?

Reply:
It should be, however the return via Thrift is typed as Any and not as List, therefore raising the following error in mypy:
![image](https://user-images.githubusercontent.com/11505309/106970511-c584c700-672b-11eb-8227-5ef68102e2ce.png)
Then I enforced to be a list regardless :smile:

Reply:
nice